### DISCUSSION TITLE
tcp connections been closed automatically with winboot in UEFI mode

### DISCUSSION THREAD

**Discussion by mtdcy** [ipxe/ipxe#947] [General]
I'm trying to install os with iSCSI with winpe, but just after boot winpe, the tcp connections will be terminated in UEFI mode. No such issue with legacy mode.

- tcp and iscsi connect success

![image](https://user-images.githubusercontent.com/16152368/233822668-c70451f0-3d6f-469f-afe9-b4e084b29394.png)

- after boot, connections shutdown

![image](https://user-images.githubusercontent.com/16152368/233822703-13ed12d5-e0f2-4e53-9768-8fdcc0a45268.png)

**Comment by NiKiZe** [comment]
Something closes you iSCSI connection during windows boot.

What does a packet dump show here?
It is kind of hard to see how iPXE is the culprit vi here.
But what nic and iPXE binary are you using?

**Reply by mtdcy** [reply]
I'm fresh to this, so:

1. how to dump a packet?

2. I built the x64 ipxe.efi from git source, and used vmware fusion to do the test, the nic is the default one. 

more information should i provide?

**Reply by mtdcy** [reply]
I use 'tcpdump -i vmenet0 -n' to dump something, will this help? 

PS: I put a 'pause' after wimboot, this dump happens after I press enter, until the connection closed.

[tcpdump.txt.zip](https://github.com/ipxe/ipxe/files/11303107/tcpdump.txt.zip)

**Reply by mtdcy** [reply]
maybe this is the right one.

[trace.pcap.zip](https://github.com/ipxe/ipxe/files/11303114/trace.pcap.zip)
